phpBB/config/default/container/services.yml

```diff
@@ -8,13 +8,16 @@ imports:
     - { resource: services_db.yml }
     - { resource: services_event.yml }
     - { resource: services_feed.yml }
+    - { resource: services_files.yml }
```

Comment:
Will cause a nice conflict with my filename here: https://github.com/phpbb/phpbb/pull/3727
I personally don't care if I have to merge those two files.